### PR TITLE
LPS-89259 JavaDocs

### PR DESCRIPTION
**Pull request by gabrielwas**

### CODE REVIEW

**Comment by gabrielwas** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
ci:test:sf

**Comment by gabrielwas** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
ci:test:sf

**Comment by gabrielwas** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
ci:test:sf

**Comment by gabrielwas** [comment]
ci:test:sf

**Comment by gabrielwas** [comment]
ci:test:relevant

File: modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/service/DEDataRecordCollectionServiceImpl.java [hunk 1]
```diff
@@ -458,7 +459,8 @@ protected DEDataEngineRequestExecutor getDEDataEngineRequestExecutor() {
 		if (_deDataRecordCollectionSaveRecordRequestExecutor == null) {
 			_deDataRecordCollectionSaveRecordRequestExecutor =
 				new DEDataRecordCollectionSaveRecordRequestExecutor(
-					deDataStorageTracker, ddlRecordLocalService);
+					deDataStorageTracker, ddlRecordLocalService,
```
Comment (jeyvison) [inline modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/service/DEDataRecordCollectionServiceImpl.java]: Sort parameters

File: modules/apps/data-engine/data-engine-api/src/main/java/com/liferay/data/engine/storage/DEDataStorageSaveResponse.java [hunk 2]
```diff
@@ -15,20 +15,42 @@
 package com.liferay.data.engine.storage;
 
 /**
+ * Response class used as a return value for the data storage save operation
+ *
  * @author Leonardo Barros
  */
```
Comment (jeyvison) [inline modules/apps/data-engine/data-engine-api/src/main/java/com/liferay/data/engine/storage/DEDataStorageSaveResponse.java]: Missing @review

**Pull request closed without merging**